### PR TITLE
Add database

### PR DESCRIPTION
**Pull request by kjarosh**

### CODE REVIEW

File: QRServer/common/utils.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def is_guest(username: str):
+    return username.endswith(' GUEST')
```
Comment (Fruktus) [inline QRServer/common/utils.py:2]: original game checks this by also comparing password hash to hash('<NOPASSWD>') but I'm not certain whether that is necessary.

File: QRServer/lobby/lobbyclient.py [hunk 2]
```diff
@@ -51,20 +52,27 @@ def _handle_qrl(self, values):
...
         password = values[2].decode('utf8')
-        if not DBConnector().user_exists(username, password):  # FIXME implement dbconnector
-            self.send(b'<L>~<BAD_MEMBER>\x00')
-            self.close()
-        elif self.lobby_server.username_exists(username):
+
+        if not utils.is_guest(username):
+            user_id = connector().authenticate_member(username, password)
+            if user_id is None:
+                log.info('Player {} tried to connect, but failed to authenticate'.format(username))
```
Comment (Fruktus) [inline QRServer/lobby/lobbyclient.py:59]: Is loglevel Info necessary for this? I'd say more of a debug

**Review by Fruktus — CHANGES_REQUESTED** [review]
Right now we don't have a method for registration, so if we merge full authentication we'll lose the ability to use member features until we implement it. I'd rather add registration simultaneously, even if hardcoded or from some config file

**Comment by Fruktus** [comment]
We could solve the issue of lack of registration by adding command line parameter that would disable authentication when specified (enabled by default)

File: QRServer/lobby/lobbyclient.py (hunk 2 above)
Comment (kjarosh) [reply QRServer/lobby/lobbyclient.py]: sure, changed to `debug`

File: QRServer/common/utils.py (hunk 1 above)
Comment (kjarosh) [reply QRServer/common/utils.py]: sure, I've added it

**Review by Fruktus — APPROVED** [review]

**Pull request merged**